src/test/groovy/pl/allegro/tech/discovery/consul/recipes/watch/ConsulWatcherTest.groovy

```diff
@@ -270,33 +316,25 @@ class ConsulWatcherTest extends Specification {
 
     def "should not invoke callbacks for disposed watch"() {
         given:
-        def first = 0
-        def second = 0
+        def counter = 0
```

Comment:
Why do you change this test?

Reply:
this was a flaky test, were race condition were happening